rules/windows/defense_evasion_hide_encoded_executable_registry.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/11/25"
+maturity = "production"
+updated_date = "2020/11/25"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies registry write modification to hide an encoded Portable Executable. This could be indicative of adversary
+defense evasion by avoiding storing malicious content directly on disk.
```

Comment:
```suggestion
Identifies registry write modifications to hide an encoded portable executable. This could be indicative of adversary
defense evasion by avoiding the storing of malicious content directly on disk.
```